scripts/docker/web/elektrad-demo/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# same as `elektrad`, but with a KDB config set up (for demo)
+
+FROM elektra/web-base:latest
+
+WORKDIR /home/elektra
+
+# prepare demo environment
+
+# mount copy of /etc/hosts to user/hosts
+
+# mount as root user
+RUN kdb mount --with-recommends hosts user/hosts hosts
+
+# then switch to elektra user
+USER elektra
+RUN mkdir /home/elektra/.config
+RUN cp /etc/hosts /home/elektra/.config/hosts
+
+# create user/app structure
+RUN kdb set user/app
+
+RUN kdb set user/app/peers
+RUN kdb set user/app/peers/#0 "192.168.0.43"
+RUN kdb setmeta user/app/peers/#0 check/validation "^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$"
+RUN kdb setmeta user/app/peers/#0 check/validation/message "invalid ip address"
+RUN kdb setmeta user/app/peers/#1 check/validation "^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$"
+RUN kdb setmeta user/app/peers/#1 check/validation/message "invalid ip address"
+RUN kdb set user/app/peers/#1 "192.168.0.44"
```

Comment:
what are the ip addresses used here?

Reply:
they are just an example for the demo, they have no meaning

Reply:
in that case: you could provide a file containing the demo data and mount that inside the container. would reduce image layers by a lot and is probably easier to maintain in the long run (maybe you can even use something that is around in the repository for tests)